exemplar/src/main/kotlin/com/squareup/exemplar/HelloWebAction.kt

```diff
@@ -1,29 +1,29 @@
 package com.squareup.exemplar
 
-import misk.web.Get
-import misk.web.PathParam
-import misk.web.QueryParam
-import misk.web.RequestHeaders
-import misk.web.ResponseContentType
+import misk.web.*
```

Comment:
Probably want to not do start imports

Reply:
hmm that's not good! I'll look into my intellij did this. guess I should actually review this myself :)

Reply:
Fixed the issue with the sq formatter. Now I just need to get access rights to push my branch :)

Reply:
https://github.com/square/java-code-styles/pull/52